src/odb/src/defin/definReader.cpp

```diff
@@ -1890,8 +1890,7 @@ dbChip* definReader::createChip(std::vector<dbLib*>& libs,
       _logger->error(utl::ODB, 250, "Chip does not exist");
     }
   } else if (chip != nullptr) {
-    fprintf(stderr, "Error: Chip already exists\n");
-    return nullptr;
+    _logger->error(utl::ODB, 250, "Chip already exists");
```

Comment:
@titan73 If you look at line 1890, you'll see that the message ID 250 is already used. You need to set the ID to a different value. 251 should work, as mentioned in the build error message.